lib/Puppeteer.js

```diff
@@ -37,7 +37,7 @@ module.exports = class {
   }
 
   /**
-   * @param {!(Launcher.BrowserOptions & {browserWSEndpoint: string, transport?: !Puppeteer.ConnectionTransport})} options
+   * @param {!(Launcher.BrowserOptions & {browserWSEndpoint: string, browserUrl: string, transport?: !Puppeteer.ConnectionTransport})} options
```

Comment:
browserUrl?

The questionmark makes it optional.

Reply:
Should we make both `browserWSEndpoint` and `browserUrl` optional then? You should pass one or another, but not both.